src/test/java/uk/ac/ebi/eva/test/utils/GenotypedVcfJobTestUtils.java

```diff
@@ -191,4 +191,10 @@ public static File getInputFile() {
     public static File getMockVep() {
         return getResource(MOCK_VEP);
     }
+
+    public static String getDefaultOpencgaHome() {
+        return System.getenv("OPENCGA_HOME") != null ?
+                System.getenv("OPENCGA_HOME") : "./src/main/resources/opencga/";
```

Comment:
It is preferred to use the `getResource` method for tests instead of regular relative paths due to potential issues with the classpath.